src/main/kotlin/calculator/Expression.kt

```diff
@@ -0,0 +1,43 @@
+package calculator
+
+import calculator.Expression.Companion.isEven
+
+class Expression(private val input: Input) {
```

Comment:
```suggestion
class Expression(input: Input) {
```
프로퍼티로 관리하지 않아도 괜찮을거 같아요!

Reply:
아 그렇네요 ~ 
아직 프로퍼티와 생성자 인자 구분이 어렵네요 ~